deepinv/datasets/patch_dataset.py

```diff
@@ -7,28 +7,31 @@ class PatchDataset(data.Dataset):
 
     :param torch.Tensor imgs: Tensor of images, size: batch size x channels x height x width
     :param int patch_size: size of patches
-    :param callable: data augmentation. callable object, None for no augmentation.
+    :param callable transforms: data augmentation. callable object, None for no augmentation.
+    :param tuple shape: shape of the returned tensor. None returns original C x H x W
```

Comment:
"None returns original C x H x W" I find this comment a bit confusing. We have H=W=patch_size right? Maybe you could write "None returns C x patch_size x patch_size"?

Reply:
Changed that!